cms/jinja2/templates/components/streamfield/glossary_section_block.html

```diff
@@ -4,22 +4,16 @@
 {%- set show_all_label = _("Show all") -%}
 {%- set hide_all_label = _("Hide all") -%}
 
-<section id="{{ value.title|slugify() }}">
-
-    <h2 class="ons-u-fs-l">{{ value.title }}</h2>
-
-    {# fmt:off #}
-    {{
-        onsAccordion({
-            "id": "accordion-" ~ block_id,
-            "allButton": {
-                "open": show_all_label,
-                "close": hide_all_label
-            },
-            "itemsList": formatted_glossary_terms,
-        })
-    }}
-    {# fmt:on#}
-
-</section>
+{# fmt:off #}
+{{
+    onsAccordion({
+        "id": "accordion-" ~ block_id,
+        "allButton": {
+            "open": show_all_label,
+            "close": hide_all_label
+        },
+        "itemsList": formatted_glossary_terms,
+    })
+}}
+{# fmt:on#}
```

Comment:
```suggestion
{{-
    onsAccordion({
        "id": "accordion-" ~ block_id,
        "allButton": {
            "open": show_all_label,
            "close": hide_all_label
        },
        "itemsList": formatted_glossary_terms,
    })
-}}
{# fmt:on #}
```

Reply:
Updated - curious to know the benefit of `{{`- vs `{{` here if both work?